cluster/vmpreset-windows.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachinePreset
+metadata:
+  name: windows-server-2012r2-preset
+  selector:
+    matchLabels:
+      os-short-id: win2k12r2
+spec:
+  terminationGracePeriodSeconds: 0
+  domain:
+    cpu:
+      cores: 2
+    resources:
+      requests:
+        memory: 512M
```

Comment:
Is this amount sufficient?

Reply:
@fabiand  its taken from the windows vm example.
maybe we should add another similar example  just for the preset use (otherwise applying the existing example with short-id and this preset will result in a conflict)

Reply:
I think we need two gig: https://technet.microsoft.com/de-de/library/dn383626(v=ws.11).aspx.

I took 512 since it is the minimum amount needed for booting it.

Reply:
+1 for 2 gigs